### DISCUSSION TITLE
Chrome extension

### DISCUSSION THREAD

**Discussion by jasonLaster** [RecordReplay/devtools#3783] [Ideas]
Was just on a call with @nicolaerusan and he suggested that Replay have a chrome extension, so all the credit goes to him.

### Problem:
People use Chrome as their default browser, so reducing the friction for record a replay with the Replay browser would be helpful

### Use Cases:
1. clicking the extension, launches Replay with that current page
2. viewing a GitHub issue or PR inserts a button that suggests recording a Replay (Loom does something similar)

**Comment by ryanjduffy** [comment]
I had the same thought for 1. in our call today. I think there's definitely some merit here!

**Comment by jonbell-lot23** [comment]
100%. This is something that would have a big impact on a common pain point, and something I think we should do.

**Comment by jasonLaster** [comment]
> Thinking about it more, when people in research would say "Oh, it's Firefox?" and that would often lead to "Can you do Chrome?" and then eventually land on "It seems like it would add friction to have to load a new browser" most people would say, unprompted, "could it be a Chrome extension?"

> Obviously the amazing thing would be for the extension to do all the Replay magic. But it's still pretty great to do what's described in the ticket: launch Replay to the URL you're on.

**Comment by jonbell-lot23** [comment]
Hey @jaril I remember you made this extension and we know it's something people would appreciate.  What's the state of the extension and what would it take to get out at a beta level?

**Reply by jaril** [reply]
Slated for Q2! https://github.com/RecordReplay/ops/issues/497
